chart/infra-server/templates/monitoring/servicemonitors.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: argo-workflows
+  namespace: argo
+spec:
+  endpoints:
+  - port: metrics
+  selector:
+    matchLabels:
+      app: workflow-controller
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: infra-server
+  namespace: infra
+spec:
+  endpoints:
+  - port: metrics
+    scheme: https
```

Comment:
Should this be `http`?

Reply:
No: https://github.com/stackrox/infra/blob/master/server/server.go#L99-L105 .

Reply:
It's a TLS endpoint with a self-signed certificate.

Reply:
Ah, makes sense. I'll unblock the PR, but would it be worth configuring the CA in the `tlsConfig`? I can't remember if we have that available on the cluster

Reply:
Probably, but may be a bigger story.